src/renderer/views/Watch/Watch.js

```diff
@@ -409,7 +413,13 @@ export default Vue.extend({
 
             if (typeof startTimestamp !== 'undefined') {
               const upcomingTimestamp = new Date(result.videoDetails.liveBroadcastDetails.startTimestamp)
-              this.upcomingTimestamp = upcomingTimestamp.toLocaleString()
+              const timestampOptions = {
+                month: 'long',
+                day: 'numeric',
+                hour: 'numeric',
+                minute: '2-digit'
```

Comment:
I think you should include year as well. Example: https://www.youtube.com/watch?v=O3cCYok_ukk (this video is to be published in 2024)

Reply:
Thanks for the video.

I've added a conditional so year is displayed if the video isn't published in current year